cwagner2325/WiseInbox
# Scrum 15 cw

**Changes Made**
- Added an Express.js backend with necessary routes and middleware. 
- Implemented server logic to handle incoming requests.

**How to Test**
- Ensure the frontend and backend dependencies are installed.
- Start the backend server using node BackEnd/index.js.
- Start the React Native front end using npx react-native start.
- Verify that the frontend can interact with the backend endpoints.
